conduct/COC_TEAM_CHARTER.md

```diff
@@ -0,0 +1,92 @@
+This document defines how the OpenJS Foundation implements its code of conduct.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Team Charter
+
+The Cross Project Council (CPC) delegates enforcement of the [Code of Conduct][CoC] to the Code of Conduct Team (CoC Team) through this Code of Conduct Team Charter. This charter defines the composition and roles of the CoC Team, along with CoC team responsibilities and operations.
+
+## Responsibilities
+
+[TODO: add links]
+
+The CoC Team's responsibilities are to:
+
+- Handle code of conduct [incident reports][incident report].
+- Handle code of conduct [escalation requests][escalation].
+- Handle code of conduct [appeals][appeal].
+- Keep the CPC up to date of cases requiring the CPC's attention.
+- Provide regular transparency reports to the CPC.
+- Define, document, and maintain all processes necessary to its operations, notably the [CoC Process for Incident Management](COC_PROCESS_FOR_INCIDENT_MANAGEMENT.md).
+- Work with OpenJS Foundation staff to develop infrastructure and processes necessary to comply to the requirements of the [code of conduct policy][CoC Policy], notably when it comes to record keeping, confidentiality, and transparency.
+- Uphold this charter, the [code of conduct][CoC], and the [code of conduct policy][CoC Policy].
+
+## CoC Team Structure
+
+The CoC Team is composed of five (5) primary members and a variable number of alternate community members.
+
+### Primary members
+
+The five primary members are:
+
+* The OpenJS Foundation Executive Director
+* The OpenJS Foundation Communications Manager
+* Three (3) community members
+
+The name and email address of the primary members are documented in the [code of conduct][primary members].
+
+### Member selection
+
+The CPC will hold an annual selection process in order to select primary and alternate community members.
+
+[TODO:
+- candidates can self-nominate
+- candidates can self-nominate for alternate or primary role
+- candidate must meet eligibility requirements
+- CPC must work towards creating a CoC Team that is diverse
+- CPC member can block a candidate if they can motivate the reason why
+]
+
+### Eligibility
+
+Primary and alternate members of the CoC Team must: 
+
+* Commit to uphold the Foundation's [Code of Conduct][CoC]
+* Commit to uphold the Foundation's [Code of Conduct Policy][CoC Policy]
+* Commit to uphold this charter
+* Be willing to treat every accused individuals in the same way regardless of the impact of their contributions to open source
```

Comment:
I don't know if this is worth including, as it seems more ideological rather than substantial. It's the only section of this document that is written that way.

For clarity, what I'd consider substantial in this section are things like "cannot exceed 20% representation from the same employer".

Reply:
Following our conversation on Friday Jan 26, we agreed to keep the criteria mentioned above. I've also included an additional provision to limit the number of primary members with the same employer in https://github.com/openjs-foundation/cross-project-council/pull/1135/commits/7f817b4aa479dfaa00bdf2b9e42e5fc1e944c53a.

@bnb please add any other criteria you had in mind and resolve the conversation when you're good.